src/components/Error/Error.tsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { FC } from 'react';
+
+type Props = {
+  hasError: string,
```

Comment:
The type of 'hasError' should be boolean instead of string since it checks for the presence of an error. 🧐

Reply:
let's fix it